lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -120,6 +116,8 @@ private YarnContainerManager(
     this.yarnProxyUser = yarnProxyUser;
     this.rackNameFormatter = rackNameFormatter;
     this.trackingUrl = trackingURLProvider.getTrackingUrl();
+    this.host = trackingURLProvider.getHost();
+    this.port = trackingURLProvider.getPort();
```

Comment:
It would be a bit more readable to keep a reference to the `TrackingURLProvider` only instead of having three member variables for host, port, and URL